crates/workspace/src/item.rs

```diff
@@ -178,6 +178,33 @@ impl TabContentParams {
     }
 }
 
+pub struct TabTooltipContent(Box<dyn for<'a> Fn(&'a mut WindowContext) -> AnyView + 'static>);
```

Comment:
This type seems a little odd to me. We don't really do anything like this anywhere else, to my knowledge.